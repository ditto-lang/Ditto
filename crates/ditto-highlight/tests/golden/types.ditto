module Test exports (
    Foo,
    Bar(..),
);


type alias T = SomeModule.T(Int);

type alias Identity(a) = (a) -> a;
