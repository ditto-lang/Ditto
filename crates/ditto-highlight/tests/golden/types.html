<KeywordImport>module</KeywordImport> <Namespace>Test</Namespace> <KeywordImport>exports</KeywordImport> <Bracket>(</Bracket>
    <Type>Foo</Type><Delimiter>,</Delimiter>
    <Type>Bar</Type><Bracket>(</Bracket><Symbol>..</Symbol><Bracket>)</Bracket><Delimiter>,</Delimiter>
<Bracket>)</Bracket><Delimiter>;</Delimiter>


<Keyword>type</Keyword> <Keyword>alias</Keyword> <Type>T</Type> = <Namespace>SomeModule.</Namespace><Type>T</Type><Bracket>(</Bracket><Type>Int</Type><Bracket>)</Bracket><Delimiter>;</Delimiter>

<Keyword>type</Keyword> <Keyword>alias</Keyword> <Type>Identity</Type><Bracket>(</Bracket><TypeVariable>a</TypeVariable><Bracket>)</Bracket> = <Bracket>(</Bracket><TypeVariable>a</TypeVariable><Bracket>)</Bracket> <Type>-></Type> <TypeVariable>a</TypeVariable><Delimiter>;</Delimiter>
